Riot/ViewController/RoomViewController.m

```diff
@@ -1583,7 +1580,8 @@ - (void)refreshPreviewHeader:(BOOL)isLandscapeOriented
 {
     if (previewHeader)
     {
-        if (isLandscapeOriented)
+        if (isLandscapeOriented
+            && [GBDeviceInfo deviceInfo].family == GBDeviceFamilyiPad)
```

Comment:
The checked condition is wrong, it should be ([GBDeviceInfo deviceInfo].family != GBDeviceFamilyiPad) no?